go/private/bzl_format.bzl

```diff
@@ -38,8 +38,7 @@ py_binary(
 
 
 _bzl_format_yapf_repository = repository_rule(
-    implementation = _bzl_format_yapf_repository_impl,
-    attrs = {})
+    implementation = _bzl_format_yapf_repository_impl, attrs = {})
```

Comment:
skylark-style is one key-value pair to a line, so not a fan of this re-format.  Can yapf be configured?